plugins/dynamic-home-page/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@internal/plugin-dynamic-home-page",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin",
+    "pluginId": "dynamic-home-page",
+    "pluginPackages": [
+      "@internal/plugin-dynamic-home-page"
+    ]
+  },
+  "sideEffects": false,
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "lint": "backstage-cli package lint",
+    "postpack": "backstage-cli package postpack",
+    "postversion": "yarn run export-dynamic",
+    "prepack": "backstage-cli package prepack",
+    "dev": "backstage-cli package start",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.14.9",
+    "@backstage/core-plugin-api": "^1.9.3",
+    "@backstage/plugin-home": "^0.7.9",
+    "@backstage/plugin-search": "^1.4.15",
+    "@backstage/plugin-search-react": "^1.7.14",
+    "@backstage/theme": "^0.5.6",
+    "@material-ui/core": "^4.9.13",
+    "@mui/material": "^6.0.2",
```

Comment:
You should probably stick with the version defined in `packages/app/package.json`, I don't think both of these are needed.

Reply:
I kept material ui because the existing cards uses makeStyles. I think we can check an upgrade to MUI v5 in 1.4 when we are sure that MUI v5 works fine.